### DISCUSSION TITLE
How to make `--data` available from a file?

### DISCUSSION THREAD

**Discussion by eyllanesc** [copier-org/copier#540] [Q&A] [answered]
If I have the following configuration file:

**copier.yml**

```yaml
key1:
  type: str

key2:
  type: str
```
I know you can pass the arguments through `--data`: 

```console
copier from to --data key1="foo" --data key2="bar"
```

But in some cases you want to pass the predefined responses, for example a file: 

**config.yml**

```yaml
key1: foo
key2: bar
```

How can I do something like: `copier from to ---data-from-file config.yml`?

**Comment by eyllanesc** [comment, marked as answer]
Ooops, the solution is documented in:  https://copier.readthedocs.io/en/stable/configuring/#answers_file

**Reply by white-gecko** [reply]
How do you achieve this when initially copying a template using an answerfile? It does not appear to be the solution to the problem you describe. The stale pr #485 mentioned by @yajo could be the solution …

**Comment by yajo** [comment]
https://github.com/copier-org/copier/pull/485 is an attempt to implement that, but it seems stale now.
